models/select-all-articles.model.js

```diff
@@ -1,9 +1,16 @@
 const db = require('../db/connection')
 
 
-exports.selectAllArticles = () => {
+exports.selectAllArticles = (query) => {
+    const { sort_by, order } = query
+
+    const sortingOrder = order? order: "DESC"
+    const sortingQuery = sort_by? sort_by: 'created_at'
+    
+
+
```

Comment:
This is open to sql injection attacks. You should be checking the values to ensure they are what you expect them to be. There are various ways of doing this, using something like a list of known valid values since there are only a limited amount may be an option you can look at.